src/Decoder.c

```diff
@@ -3445,6 +3499,16 @@ static void ZydisSetEffectiveOperandWidth(ZydisDecoderContext* context,
     };
 
     ZyanU8 index = (instruction->attributes & ZYDIS_ATTRIB_HAS_OPERANDSIZE) ? 1 : 0;
+
+    if ((instruction->encoding == ZYDIS_INSTRUCTION_ENCODING_EVEX) &&
+        (instruction->opcode_map == ZYDIS_OPCODE_MAP_MAP4) &&
+        (instruction->raw.evex.pp == 0x01))
+    {
+        // EVEX encoded instructions in MAP4 must use 0x66 as the mandatory prefix AND the
```

Comment:
Might be better to have a flag for this. We usually don't hardcode table related conditions.